Tests/Fixtures/app/config/base_config.yml

```diff
@@ -1,14 +1,15 @@
 framework:
     secret:        test
-    router:        { resource: "%kernel.root_dir%/config/routing.yml" }
+    router:        { resource: "%kernel.project_dir%/app/config/routing.yml" }
     test:          ~
     translator:    { enabled: true }
     validation:    { enabled: false }
 
 # Twig Configuration
 twig:
-    debug:            "%kernel.debug%"
-    strict_variables: "%kernel.debug%"
+    debug:                "%kernel.debug%"
+    strict_variables:     "%kernel.debug%"
+    exception_controller: null
```

Comment:
if you want to keep using a NotFoundHttpException in the controller without breaking tests for Symfony 4.3 and older, the solution would be to apply this `exception_controller: null` config only on Symfony 4.4+ (because here, you disable *all* error handling on older versions as they don't have the new system)